src/Console.tsx

```diff
@@ -67,7 +67,7 @@ const ConsoleFrame: React.FC = () => {
   const input = useRef<HTMLInputElement>(null)
 
   useEffect(() => {
-    const hookedConsole = Hook(window.console, (log: Message) => setLogs(logs => logs.concat(Decode(log))))
+    const hookedConsole = Hook(window.console, (log: Message) => setLogs((logs) => logs.concat(Decode(log))))
```

Comment:
What lint rules have changed? I couldn't find it in this PR

Reply:
I think it was the bunch of updates coming from dependabot
Prettier changed, for one.